mantidproject/mantid
# Add new VULCAN geometry

Add the new instrument geometry created in [mantid-geometry 177](https://github.com/mantidproject/mantidgeometry/pull/177/). 

**To test:**

```python
ws = Load("/SNS/VULCAN/IPTS-26807/nexus/VULCAN_192242.nxs.h5")
LoadInstrument(Workspace=ws, InstrumentName="VULCAN", RewriteSpectraMap="True")  # must be string
ConvertUnits(InputWorkspace=ws, OutputWorkspace=ws, Target="dSpacing", EMode="Elastic")
```
then color-fill-plot and see that the peaks are (roughly) aligned

*There is no associated issue.*

The version into `ornl-next` is #30734.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
